File: Next with CosmosDB/server/Basketball/Microservice/Infrastructure/BasketballResponse.cs
using System.Text.Json.Serialization;

namespace NBA.Infrastructure
{
    public class BasketballResponse<T>
    {
        [JsonIgnore]
        public int Code { get; internal set; }
        public string Message { get; internal set; }
        public T PayLoad { get; internal set; }
    }
}

File: Vue with MongoDB/server/Basketball/Microservice/src/main/java/com/nba/microservice/infrastructure/BasketballResponse.java
package com.nba.microservice.infrastructure;

import com.fasterxml.jackson.annotation.JsonIgnore;

public class BasketballResponse<T> {

    @JsonIgnore
    private int code;

    private String message;

    private T payLoad;

    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public T getPayLoad() {
        return payLoad;
    }

    public void setPayLoad(T payLoad) {
        this.payLoad = payLoad;
    }
}